packages/components/buttons/flat-button/src/flat-button.js

```diff
@@ -67,47 +68,45 @@ export const FlatButton = (props) => {
     // we fall back to `isDisabled`
     disabled: props.isDisabled,
   };
-
+  const theme = useTheme();
+  const overwrittenVars = {
+    ...vars,
+    ...theme,
```

Comment:
so consumer aren't allow to override the theme at all?

Reply:
They are yes, with the `ThemeProvider`.

This change here is only about how we access the `theme` object internally by being explicit.